src/go/nerdctl-stub/command_handlers.go

```diff
@@ -22,15 +23,129 @@ func imageBuildHandler(c *commandDefinition, args []string) (*parsedArgs, error)
 		// input is a URL
 		return &parsedArgs{args: args}, nil
 	}
-	newPath, cleanups, err := filePathArgHandler(args[0])
+	newPath, cleanups, err := argHandlers.filePathArgHandler(args[0])
 	if err != nil {
-		for _, cleanup := range cleanups {
-			cleanupErr := cleanup()
-			if cleanupErr != nil {
-				log.Printf("Error cleaning up: %s", cleanupErr)
-			}
-		}
+		runCleanups(cleanups)
```

Comment:
This code is now silently ignoring cleanup errors. Is that intentional?

Reply:
Good catch, fixed (and tested) in fc75f1d6f472f89f04379905a899956a1bd9557c.